contracts/rollup.sol

```diff
@@ -191,11 +190,9 @@ contract RollupHelpers is RollupSetup {
             Types.Batch memory batch = batches[i];
 
             // calculate challeger's reward
-            uint256 _challengerReward = (batch.stakeCommitted.mul(2)).div(3);
+            uint256 _challengerReward = (STAKE_AMOUNT.mul(2)).div(3);
             challengerRewards += _challengerReward;
-            burnedAmount += batch.stakeCommitted.sub(_challengerReward);
-
-            batches[i].stakeCommitted = 0;
+            burnedAmount += STAKE_AMOUNT.sub(_challengerReward);
 
             // delete batch
             delete batches[i];
```

Comment:
This is a little bit strange to me. Why we are setting stakeCommitted = 0 before we are deleting the whole batch?
This is why I consider stakeCommitted does not act as the state of a batch being slashed is not.

Reply:
yeah, it does seem strange to me too, I am sure there was a reason for that which I should have commented, but for now, let's consider it a mistake.